.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+<!-- Fill-in description here -->
+
+## Dependencies
+
+<!-- List here any related PRs with links, that need to be pulled also for testing -->
+n/a
+
+## Checklist
+
+If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.
+
+* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
+* [ ] Does this PR require product documentation?
+  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
+* [ ] Does this PR require a product release notes entry?
+  * [ ] If so, fill in "Release Note Text" in the JIRA.
+* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
+  * [ ] If so, make sure it is described in the JIRA ticket.
+* QE requirements (check 1 from the list):
+  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
+  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
+  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).
```

Comment:
I would split more each step especially for a checklist exposed to external contributors.

```suggestion
* [ ] Is this PR backed with a JIRA ticket? 
If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
  * [ ] PR title prefixed
  * [ ] Has user facing changes 
* [ ] Does this PR require product documentation?
If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. 
Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
  * [ ] Label "documentation" has been added
  * [ ] Description is ready for docs
  * [ ] Configuration steps are documented
* [ ] Does this PR require a product release notes entry?
If so, fill in "Release Note Text" in the JIRA.
  * [ ] Release note text has been added
* Is there anything else the QE team should know before testing? 
E.g: configuration changes, environment setup, etc.
  * [ ] Additionnal testing info described in the JIRA ticket
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise
  * [ ] Regression tests only 
  (e.g. refactoring with no user-facing change)
  * [ ] No QE 
  (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team)
```

It's more wordy but ensure each item is done. WDYT ?

Reply:
hmm could we first give a try with the less detailed one, and add items just if we feel the need? (It's just that ... I don't want to have the feeling to complete an annual tax form for every PR 😵‍💫 😁)